diff --git a/node_modules/@mdx-js/mdx/index.js b/node_modules/@mdx-js/mdx/index.js
index b5ea9a8..20bea05 100644
--- a/node_modules/@mdx-js/mdx/index.js
+++ b/node_modules/@mdx-js/mdx/index.js
@@ -22,7 +22,7 @@ function createMdxAstCompiler(options = {}) {
 function createCompiler(options = {}) {
   return createMdxAstCompiler(options)
     .use(options.rehypePlugins)
-    .use(minifyWhitespace, {newlines: true})
+    // .use(minifyWhitespace, {newlines: true})
     .use(mdxHastToJsx, options)
 }
 
